adm_program/modules/dates/dates_function.php

```diff
@@ -387,7 +387,7 @@
             // Kategorie fuer Terminbestaetigungen einlesen
             $sql = 'SELECT cat_id
                       FROM '.TBL_CATEGORIES.'
-                     WHERE cat_name_intern = \'CONFIRMATION_OF_PARTICIPATION\'';
+                     WHERE cat_name_intern LIKE \'EVENTS\'';
```

Comment:
only use `LIKE` if you search for `%XXX%` or something else
`WHERE cat_name_intern = \'EVENTS\'';`